ProgrammingTools/Write_ChckOutLst.m

```diff
@@ -53,15 +62,15 @@
         ModName = 'Morison';
     case 'SeaState'
         ModName = 'SeaSt';
-    case 'FAST.Farm'
+    case 'FAST_Farm'
```

Comment:
Did you rename  the sheet in OutListParameters.xlsx? Mine still says `FAST.Farm`, not `FAST_Farm`... but I've been dealing with a lot of merge conflicts on that file recently, so it's possible I missed a rename there.

Reply:
I did rename the worksheet, but I didn't really know what to do there. There are lots of inconsistencies with this module name. The directory structure of the repository uses fast-farm, the module name uses FAST_Farm, and the official name is `FAST.Farm`. We could also simply add a line in the matlab script that deals with this particular case. 

I'm also thinking of suggesting to get rid of the Excel file, and splitting it into multiple csv file, that can actually be tracked by git. I'll bring that up at the next meeting.